content/4_Gold/Divis.mdx

```diff
@@ -51,6 +51,14 @@ If you've never encountered any **number theory** before, AoPS is a good place t
 	>
 		good book
 	</Resource>
+	<Resource
+		source="Aditya Khurmi@AoPS User: Wizard_32"
```

Comment:
Why isn't this source `MONT@Modern Olympiad Number Theory` like the others?

Reply:
The book leads you to discover theorems rather than tell you. AoPS intro to NT was a very good book. This is basically like it except more advanced.

Reply:
But aren't they the same books?

Reply:
No, this one is free

Reply:
I meant that you linked the book three times, but this link has a different title from the rest

Reply:
I linked it in the first one because the top resources were for good overall NT books. I linked a few chapters for the divisor resources section. And then more modular arithmetic, I linked a couple chapters in the resource section.

Reply:
I understand why you linked it, but why is the ***title*** different even though it's the same book?

Reply:
should probably just remove this occurrence?

Reply:
I mean its a good book for overall Number Theory

Reply:
> I understand why you linked it, but why is the _**title**_ different even though it's the same book?

Also I changed the title in the second occurrence because I just thought I should make it what they should do in the book cause I see it in other books in the guide.

Reply:
> I mean its a good book for overall Number Theory

Well, I'm sure you would see that if you clicked on the link under "Resources" ... I don't think that necessitates linking it twice.

I think @dolphingarlic is referring to the source?

Reply:
ok, so should I get rid of it in the top table?

Reply:
yes

Reply:
done